pkg/config/load.go

```diff
@@ -159,13 +159,24 @@ func readCiOperatorConfig(configFilePath string, info Info) (*cioperatorapi.Rele
 		return nil, fmt.Errorf("failed to load ci-operator config (%w)", err)
 	}
 
+	for i := range configSpec.Tests {
```

Comment:
The `readCiOperatorConfig` function is critical and used all over the code base, we need some unit tests to cover the new behavior you're introducing.